iocore/net/quic/QUICBidirectionalStream.cc

```diff
@@ -418,7 +418,7 @@ QUICBidirectionalStream::generate_frame(uint8_t *buf, QUICEncryptionLevel level,
   uint64_t len           = 0;
   IOBufferReader *reader = this->_write_vio.get_reader();
   if (!pure_fin) {
-    uint64_t data_len = reader->block_read_avail();
+    uint64_t data_len = reader->read_avail();
```

Comment:
Oh, no, `IOBufferReader::read_avail()` could be so slow. Can we use `IOBufferReader::is_read_avail_more_than(int64_t size)`?